gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/05_add_environment_id.yml

```diff
@@ -2,6 +2,7 @@ databaseChangeLog:
   - changeSet:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.